test/controllers/recipes_controller_test.rb

```diff
@@ -0,0 +1,37 @@
+require "test_helper"
+
+describe RecipesController do
+  it "gets root path successfully" do
+    get root_path
+    must_respond_with :success
+  end
+
+  describe "index" do
+    it "should get a list of recpies" do
+      VCR.use_cassette("muncher") do
+        get recipe_results_path
+        recipes = EdamamApiWrapper.list_recipes("Nooch")
```

Comment:
Why are you mixing using the **Wrapper** in the controller test.  In this file you should only be testing the controller.  

Instead this test should look like: 
```ruby
    it "should get a list of recpies" do
      VCR.use_cassette("muncher") do
        get recipe_results_path, params: {q: "Nooch"}
      end
      must_respond_with :ok
    end
```